circle.yml

```diff
@@ -387,6 +387,25 @@ jobs:
             prague/eip7002_el_triggerable_withdrawals
             prague/eip7251_consolidations
             prague/eip7685_general_purpose_el_requests
+      - download_execution_spec_tests:
```

Comment:
I think this unpacks them into the same `fixtures` subfolder as previous `pectra-devnet-4` download?

Reply:
Yes. If an issue clear the directory before unpacking in the download step